### PR TITLE
Add readme.

### PR DESCRIPTION
**Pull request by Mehrnaz-Charkhchi**
[ENSWBSITES-2665](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2665)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Ensembl Web Resolver
+Resolver resolves and redirects external urls to Ensembl ones
```
Comment (azangru) [inline README.md:2]: > redirects external urls to Ensembl ones

I don't think they are external urls though. `resolver.ensembl.org/id/ENSG00000139618` isn't an external url; it is resolver's url.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ 
+ `$ git clone https://github.com/Ensembl/ensembl-web-resolver.git`
+ `$  cd ensembl-web-metadata-api`
+ `$ mv sample-env .env`
+ `$  docker-compose -f docker-compose.yml up`
+ 
+ ### Deploy the app and run docker-compose:
+ Some urls that are available after deployment on your local machine:
+ http://localhost:8001/api/id/ENSG00000127720
+ http://localhost:8001/api/id/ENSG00000127720.3
```
Comment (azangru) [inline README.md:14]: Why are we adding the `/api` namespace? Wasn't it the idea that the resolver would be running on `resolver.ensembl.org/id/...`? Or will we rewrite that in ingresses?

File: README.md (hunk 1 above)
Comment (Mehrnaz-Charkhchi) [reply README.md]: I made this update.

File: README.md (hunk 2 above)
Comment (Mehrnaz-Charkhchi) [reply README.md]: You're right the only thing is that on the review branch it was /api and I forgot to remove it.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Ensembl Web Resolver
+The resolver service generates new Ensembl website urls for different features based on their stable ids, as well as other optional parameters.
+### Deploy the app and run docker-compose:
+ 
+ `$ git clone https://github.com/Ensembl/ensembl-web-resolver.git`
+ `$  cd ensembl-web-metadata-api`
+ `$ mv sample-env .env`
```
Comment (jyothishnt) [inline README.md:7]: Should we replace the search hub url in `sample-env` with beta url?

Comment (jyothishnt) [reply README.md:7]: `https://beta.ensembl.org/api/search/stable-id`
with below data as post request
{code}
{
    "stable_id": "WBGene00016374",
    "type": "gene"
}
{code}

Comment (Mehrnaz-Charkhchi) [reply README.md:7]: What do you want me to do with the payload?

Comment (Mehrnaz-Charkhchi) [reply README.md:7]: env-sample is updated

Comment (jyothishnt) [reply README.md:7]: Nothing, that payload was just for you to try out :-)  Approving now

**Review by jyothishnt — APPROVED** [review]

**Pull request merged**